packages/repeater/src/utils/NormalizeZlibDeflateTransformStream.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { NormalizeZlibDeflateTransformStream } from './NormalizeZlibDeflateTransformStream';
+import { promisify } from 'node:util';
+import { constants, createInflate, deflate, deflateRaw } from 'node:zlib';
+import { Readable } from 'node:stream';
+
+const zOpts = {
+  flush: constants.Z_SYNC_FLUSH,
+  finishFlush: constants.Z_SYNC_FLUSH
+};
+
+// TODO: replace with Readable.from once support for Node 10 is dropped
+const readableFrom = (buffer: Buffer) => {
+  const stream = new Readable();
+  stream.push(buffer);
+  stream.push(null);
+
+  return stream;
+};
+
+describe('NormalizeZlibDeflateTransformStream', () => {
+  it('should add zlib headers to raw deflate', async () => {
```

Comment:
All test cases must be under the method's suite: `NormalizeZlibDeflateTransformStream > pipe -> should do ....`